app/sidecar/base.toml

```diff
@@ -119,6 +130,9 @@ task-slots = [
 features = ["sidecar", "vlan", "auxflash", "vpd"]
 notifications = ["socket", "usart-irq", "timer"]
 
+[tasks.control_plane_agent.config]
+trusted-keys = ["../../support/fake_keys/id_ecdsa.pub"]
```

Comment:
TODO: switch to the actual support keys from https://github.com/oxidecomputer/support-piv-tokens/

Reply:
I assume you plan to have all keys/modes compiled in and then select based on the RoT providing information to the SP as to the mode (prod-release, etc. per your previous PR) with the default being to fail safe.
Maybe you're thinking of a way to compile in just the keys needed for the ultimate signed image.

Reply:
Basically!  There are two modes, based on what the RoT reports:

- Release, which sends `UnlockChallenge::EcdsaSha2Nistp256`, and the reply must match one of these keys
- Not-release, which sends `UnlockChallenge::Trivial` (so the reply is also trivial)

In other words, there's no "fake keys" mode (and I'm going to delete the `fake_keys` subfolder before moving this out of draft).  Otherwise, we have the Illusion of Security, and have to explain why we've committed private keys to the repo.

The two-mode logic is already implemented in `control-plane-agent`; it's just the actual keys that need to change.